project/src/main/webapp/index.html

```diff
@@ -19,7 +18,11 @@ <h1 style= "color: rgb(63, 63, 209);">Car Collision Map:</h1>
 src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDbskcye9gWYfKt8nHzuRaPudtpo2yr0so&region=US&language=en&callback=initMap">
 </script>
     <body onload="createMap();">
-    <div id="map"></div>
-</body>
+        <div id="map"></div>
+    </body>
+    <!-- <body onload="create_markers();">
```

Comment:
If this is no longer useful, please remove it.

Reply:
Just fix it!